### PR TITLE
updated programs and other texts for Python 3.6

### PR DESCRIPTION
**Pull request by ramlaxman**

### CODE REVIEW

File: 02-programmingBasics/02-constants.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-
-print 3455
-
-print 45.45
-
-
+# constants representation of Python
+print ("this is a constant string")
+# bare numbers are treated as strings
+<<<<<<< HEAD
```
Comment (narendrapetkar) [inline 02-programmingBasics/02-constants.py]: please resolve merge conflict and delete conflict lines.

File: 02-programmingBasics/02-variables.py [hunk 2]
```diff
@@ -5,25 +5,41 @@
...
 
+print ("Simple arithmatic with variables : ")
+print ("\tAddition : ", (var1 + var2))
+print ("\tSubstraction : ", (var1 - var2))
+print ("\tMultiplication : ", (var1 * var2))
+print ("\tDivision : ", (var1 / var2))
 
+VAR1 = 33
+print ("Same names yet different value due to Case sensitive names : %d -- %d " % (var1, VAR1))
+>>>>>>> f843d94c4c04d61508c4a52a5c6cbc9865c4079a
```
Comment (narendrapetkar) [inline 02-programmingBasics/02-variables.py]: remove conflict line.

Comment (ramlaxman) [reply 02-programmingBasics/02-variables.py]: मी issue resolve केला आहे. अजून काही problem असल्यास कळवावे.

**Comment by narendrapetkar** [comment]
Thanks for the contribution. There are following suggestions in general.
-- Please resolve merge coflicts and remove the lines that denote conflicts from all files.
	e.g. **<<<<HEAD** or **>>>>>>> <commit_id>** or **=========**
-- Make sure all files can be run directly. If done already, well n good.
	e.g. if you checkout code / download zip, all .py files should be runnable by $python3 <fileName.py>

**Comment by ramlaxman** [comment]
Done. I have taken a look at all the code that I pushed. Do let me know in case of any issue.

**Comment by narendrapetkar** [comment]
Ok. I'll merge uptill now and test at my place. although it might not be till 1-2 days. I'll try as early as possible.

**Pull request merged**